src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -2301,10 +2301,15 @@ void TLayout::layout(FretCircle* item, LayoutContext&)
     double offsetFromUpNote = (item->rect().height() - chordRect.height()
                                - (item->chord()->downNote()->pos().y() - item->chord()->upNote()->pos().y())
                                ) / 2;
-    item->setOffsetFromUpNote(offsetFromUpNote);
-    item->setSideOffset((item->rect().width() - minWidth) / 2);
+    ldata->offsetFromUpNote = offsetFromUpNote;
+    ldata->sideOffset = ((item->rect().width() - minWidth) / 2);
+
+    ldata->setbbox(item->rect().adjusted(-lw, -lw, lw, lw));
```

Comment:
What do you think about replacing `item->rect()` with `ldata->rect` in these lines?